designsafe/apps/api/notifications/tests.py

```diff
@@ -176,3 +176,58 @@ def test_webhook_interactive_post(self, get_user_model, mock_receiver):
         self.assertEqual(action_link, link_from_event)
         self.assertEqual(n.operation, 'web_link')
         self.assertEqual(response.status_code, 200)
+class NotificationsViewsTests(TestCase):
+    fixtures = ['user-data.json', 'agave-oauth-token-data.json']
+
+    def setUp(self):
+        #user = get_user_model().objects.get(username="ds_user")
+        note = Notification(
+            event_type="JOB",
+            status="SUCCESS",
+            message="Job succeeded",
+            user="ds_user",
+        )
+        note.save()
+        #self.client=Client()
```

Comment:
Try to avoid having commented code on the finished version. If the code is already in a past commit you can hunt it back by looking at the history of the file.